### ISSUE TITLE
Split window cursor repositioning when using CTRL-Z 

### ISSUE THREAD

**Issue by dreappdev** [apache/netbeans#6711]
### Apache NetBeans version

Apache NetBeans 19

### What happened

When I split (or clone) an editor window, then edit something in the file, then use CTRL-Z or SHIFT-CTRL-Z to backout a typo or redo an edit, the split (or cloned) window repositions the cursor location to where the action happened. What I expected was that the split window would remain positioned wherever I had set it. Which it does while doing the edits - but not when using the CTRL-Z. 

### How to reproduce

Open any file to edit it.
Split the editor window, either horizontal or veritical, or clone it. If you clone it, you will have to undock the cloned window to see both windows at the same time.
Position the cursor in the two panes or windows in totally different locations, making sure to not show any segment of the file in both panes.
Make an edit in one of the panes. The other pane will remain positioned where you just placed it. This is good.
Do a CTRL-Z to undo the edit. The other pane will reposition the cursor to show the area of the file where the edit was just undone. This is not good.

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

macOS Ventrua Version 13.5.1

### JDK

Java: 17.0.2; Java HotSpot(TM) 64-Bit Server VM 17.0.2+8-LTS-86

### Apache NetBeans packaging

Apache NetBeans provided installer

### Anything else

This is a repeatable issue. Happens every time.

### Are you willing to submit a pull request?

Yes

**Comment by mbien** [comment]
So you want to editors which are not in focus to not scroll if the undo action makes a modification? (because the editor in focus will scroll to the modified spot too)

**Comment by dreappdev** [comment]
Correct.Thanks, DErikOn Nov 17, 2023, at 9:14 AM, Michael Bien ***@***.***> wrote:﻿
So you want to editors which are not in focus to not scroll if the undo action makes a modification? (because the editor in focus will scroll to the modified spot too)

**Comment by jimklimov** [comment]
According to https://stackoverflow.com/questions/63065307/stop-auto-scrolling-on-a-split-or-cloned-window-in-netbeans this is broken at least since NetBeans 11

I've just double-checked that in NetBeans 8.2 this still worked correctly (editing and undo impacted the cursor only in the focused pane of a document split into two windows).
